lib/util/makeRequest.ts

```diff
@@ -82,45 +63,38 @@ const promisifiedRequest = (options: Options) =>
  * @return {Promise} promise - A promise that resolves to the responseBody or rejects to an error.
  */
 export async function makeRequest(
-  options: Options,
+  url: string,
   reportError: jsonParser.ReportError
 ): Promise<SwaggerObject> {
-  return retry(
-    async (bail, retryNr) => {
-      const { err, response, responseBody } = await promisifiedRequest(options)
-
-      if (err) {
-        const message = `Request to ${
-          options.url
-        } failed with error ${err} on retry number ${retryNr}.`
-
-        throw new Error(message)
-      }
+  return retry<SwaggerObject>(
+    async (bail, retryNr): Promise<SwaggerObject> => {
+      try {
+        const response = await fetch(url)
+        const body = await response.text()
 
-      if (options.errorOnNon200Response && response.statusCode !== 200) {
-        const msg = `StatusCode: "${
-          response.statusCode
-        }", ResponseBody: "${responseBody}."`
+        if (response.status !== 200) {
```

Comment:
are we sure its going to be a 200 ? maybe we can do > 399 ?

Reply:
why wouldn't it be 200?

Reply:
could be a redirect or god knows, was airing on the safer side